src/main/java/cubicchunks/server/ChunkGc.java

```diff
@@ -55,13 +53,6 @@ public void tick() {
 	}
 
 	private void chunkGc() {
-		for (Chunk chunk : cubeCache.getLoadedChunks()) {
-			Column column = (Column) chunk;
-			Collection<Cube> cubes = column.getLoadedCubes();
-			cubes.stream().filter(c -> !cubeMap.contains(c.getCoords())).forEach(cubeCache::unloadCube);
-			if(!cubeMap.contains(column.getX(), column.getZ())) {
-				cubeCache.unloadColumn(column);
-			}
-		}
+		cubeCache.takeOutGarbage();
```

Comment:
...

The whole point of ChunkGc being separate class was that the unloading logic was in that class.

Now you moved it to CubeCache.